pkgs/os-specific/linux/bluez/default.nix

```diff
@@ -116,7 +116,7 @@ stdenv.mkDerivation rec {
 
   meta = with stdenv.lib; {
     description = "Bluetooth support for Linux";
-    homepage = http://www.bluez.org/;
+    homepage = "http://www.bluez.org/";
```

Comment:
This should help slowly drag us forward toward https://github.com/NixOS/rfcs/pull/45 compliance, with some assistance from our robot overlords :)

Reply:
oh nice, automatic quote enforcement